src/components/github-contributors-list/github-contributors-list.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC, useEffect, useMemo, useState } from 'react';
+import ContributorsList from './contributors-list';
+import Paragraph from '../../../src/components/paragraph/paragraph';
+import { getAllContributors, GithubContributor, GithubContributorResponse } from './github-contributors-utils';
+
+interface GithubContributorsListProps {
+  organizationName: string;
+  packageName: string;
+  text?: string;
+  showContributionAmount?: boolean;
+  excludedContributorsIds?: Set<number>;
+  staticContributors?: GithubContributor[];
+}
+
+const GithubContributorsList: FC<GithubContributorsListProps> = ({
+  organizationName,
+  packageName,
+  excludedContributorsIds = new Set(),
+  staticContributors = [],
+  text = 'Thanks to all of our contributors: ',
+  showContributionAmount = false,
+}) => {
+  const [contributorsJson, setContributorsJson] = useState<GithubContributorResponse[]>();
+  useEffect(() => {
+    getAllContributors(organizationName, packageName).then(contributors => setContributorsJson(contributors));
+  }, []);
+
+  const contributors = useMemo(() => {
+    if (contributorsJson && Array.isArray(contributorsJson)) {
```

Comment:
I think this check should be in the fetch function and defaults to an empty array